cfgov/ask_cfpb/migration_scripts/migrate_knowledgebase.py

```diff
@@ -0,0 +1,281 @@
+from __future__ import unicode_literals
+
+import time
+
+from knowledgebase.models import QuestionCategory as QC
+from knowledgebase.models import Question, Audience, UpsellItem, EnglishAnswer
+from ask_cfpb.models import (
+    Answer,
+    # AnswerPage,
+    Category,
+    NextStep,
+    SubCategory)
+from ask_cfpb.models import Audience as ASK_audience
+
+
+def get_kb_statuses(ask_id):
+    q = Question.objects.get(id=ask_id)
+    return {'en': (q.get_english_answer_state() == 'APPROVED'),
+            'es': (q.get_spanish_answer_state() == 'APPROVED')}
+
+
+def create_answer_pages(queryset):
+    counter = 0
+    if queryset:
+        for answer in queryset:
+            kb_statuses = get_kb_statuses(answer.id)
+            for lang in ['en', 'es']:
+                if kb_statuses[lang]:
+                    page = answer.create_or_update_page(language=lang)
+                    revision = page.get_latest_revision()
+                    revision.publish()
```

Comment:
@rosskarchner I think we talked about this once before, but what about adding some kind of throttled Akamai flush logic? We could do this either with a simple cache value (storing the last time a flush was invoked, don't invoke a new flush if it was too recent) or something more complex like a Django model like `AkamaiCacheFlush`. One nice side effect of this would be to have somewhere to store the unique ID we get back from the Akamai API, so we'd be able to more easily track how often we flush, how long flushes take to happen, etc.

In either case we could stick this check [here](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/wagtail_hooks.py#L146) and skip the flush if we've just done one.

The downside of this approach would be that when updating a whole bunch of pages you would only flush after the first change, when really you'd want to do the last. So maybe a slightly more hacky way might be to remove the flush logic from the web request altogether; you write flush requests to the DB table and there's an async job (Jenkins) that polls that table for outstanding requests periodically.

cc @richaagarwal